### DISCUSSION TITLE
[Bug] everything is saved with "_story_" in the name

### DISCUSSION THREAD

**Discussion by crxed9** [rodit/SnapMod#168] [General]
I am not putting this in Issues, since I am not using the most recent version, but even though the Snap Download Path setting has the word "%u_snap_" in it, all snaps are saved with "_story_" in the name, even though it was not a story.

Does something need to be changed?

**Comment by NMBlanco** [comment]
Same here

**Comment by rodit** [comment]
This is expected for now, I'll change this to use the snap download path in the next update.

**Comment by rodit** [comment]
This should be fixed in 1.8.0

**Reply by crxed9** [reply]
Confirmed fixed
